pkg/version/version.go

```diff
@@ -1,7 +1,10 @@
-package app
+package version
 
 // Version contains the broker version.
 // DO NOT EDIT GENERATED FILE.
 // This file is generated by a tito tag
 // using templates/version.go.tmpl.
 var Version = "1.0.4"
+var MinAPBVersion = "1.0"
+var MaxAPBVersion = "1.0"
```

Comment:
We might consider having an `apbversion.go` that contains the 2 values. Especially if they aren't filled in by tito.

So keep new `version` package. Move `version.go` like you did. Keep version.go just Version. Then add an `apbversion.go`.